crates/viewer/re_space_view_spatial/src/proc_mesh.rs

```diff
@@ -28,15 +28,25 @@ pub enum ProcMeshKey {
     ///
     /// The resulting mesh may be scaled to represent spheres and ellipsoids
     /// of other sizes.
-    Sphere { subdivisions: usize },
+    Sphere {
+        /// Number of triangle subdivisions to perform to create a finer, rounder mesh.
+        subdivisions: usize,
+
+        /// If true, then when a wireframe mesh is generated, it includes only
+        /// the 3 axis-aligned “equatorial” circles, and not the full triangle mesh.
+        axes_only: bool,
```

Comment:
This could maybe be better as an `enum`

Reply:
Perhaps, but it doesn't matter because this is an implementation detail that we can freely change, no?

Reply:
All source code matters, not just public APIs :)

But I'm not sure an enum here makes it that much nicer, it was just a note for a possible improvement.

Reply:
Yes, what I meant is: I don't think adding an enum would make any concrete improvement, and because this is not public we don't care about whether we need to change it to an enum later.

I don't think an enum will help in this case because, being a field of an enum, it is _always_ accompanied by the field name (in literal syntax or pattern syntax), so you don't have the "what does `do_thing(input, false, true)` mean?” problem. All the enum would do is add more boilerplate to type, I think. And if this got much more complex, then it might make more sense as separate variants of `ProcMeshKey` instead of a nested enum. Better to write the simplest version, and revise it later as needed.